index.html

```diff
@@ -129,109 +146,67 @@ <h2>Software Developer</h2>
-            <p class="text">
-              A daily selection of privately personalized reads; no accounts or
-              sign-ups required. has been the industry's standard dummy text
-              ever since the 1500s, when an unknown printer took a standard
-              dummy text.
+          <div >
+            <p class="popText">
+              Lorem Ipsum is simply dummy text of the printing and typesettingry. Lorem Ipsum
+              has been the industry's standard dummy text ever since the 1500s,
+              when an unknown Lorem Ipsum is simply dummy text of the printing
+              and typesetting industry. Lorem Ipsum has been the industry's
+              standard dummy text ever since the 1500s, when an unknown printer
+              took a galley of type and scramble.
             </p>
-            <ul class="lang">
-              <li>css</li>
-              <li>html</li>
-              <li>bootstrap</li>
-              <li class="ruby">Ruby</li>
-            </ul>
-            <button class="button" type="button">See Project</button>
           </div>
-        </li>
+          <ul class="plang">
+            <li class="pop-li">html</li>
+              <li class="pop-li">css</li>
+              <li class="pop-li">Ruby on rails</li> 
+                <li class="pop-li unique">Github</li>
+          </ul>
 
-        <li class="reverse-row card-sec">
-          <div class="placeholder1">
-            <img
-              class="image1"
-              src="./placeholder.png"
-              alt="image placeholder"
-            />
-          </div>
-          <div class="card1">
-            <h2 class="post">Multi-Post Stories</h2>
-            <p class="text">
-              A daily selection of privately personalized reads; no accounts or
-              sign-ups required. has been the industry's standard dummy text
-              ever since the 1500s, when an unknown printer took a standard
-              dummy text.
-            </p>
-            <ul class="lang">
-              <li>css</li>
-              <li>html</li>
-              <li>bootstrap</li>
-              <li class="ruby">Ruby</li>
-            </ul>
-            <button class="button" type="button">See Project</button>
+          <ul class="plangmobile">
+            <li class="pop-li">html</li>
+              <li class="pop-li">Ruby on rails</li> 
+                <li class="pop-li unique">css</li>
+          </ul>
+          <div class="buttonsPop">
+            <button type="button" class="seeProject1">
+              see project<img
+                src="icons/live.png"
+                alt="live icon"
+                class="liveIcon"
+              />
+            </button>
+            <button type="button" class="seeProject1">
+              see source
+              <img
+                src="icons/smallGitHub.png"
+                alt="github link"
+                class="liveIcon"
+              />
+            </button>
```

Comment:
https://github.com/Mutalenic/Portfolio/blob/3a3460bef3e4dd545efd3e99791e1191e9004584/index.html#L145-L210

- [ ] As required :point_down: 

![image](https://user-images.githubusercontent.com/57421823/151015393-2368bdbc-0f28-46f4-9357-b17c3811e52d.png)

As seen above, all the projects section is created dynamically with the information from the projects array (all the HTML of the section is created when the page loads). This requires changes in existing code - already created HTML code for projects list should be removed and instead it should be added via JavaScript.

This means you can use the `cardsinfo ` array and logic that you already have, to populate the projects section of the main page entirely with javascript!

If you are struggling or don't know how to do so, here is an [example](https://github.com/olawale-o/portfolio-mobile-version/pull/15/files#diff-0eb547304658805aad788d320f10bf1f292797b5e6d745a3bf617584da017051R95)

 - All those cards information should be stored into an array of objects then create a function to render through the array and create dynamically your HTML with the information from the object array, the works section must stay empty